public/app/features/alerting/unified/components/DynamicTable.tsx

```diff
@@ -125,9 +125,8 @@ export const DynamicTable = <T extends object>({
                 <div className={cx(styles.cell, styles.expandCell)}>
                   <IconButton
                     aria-label={`${isItemExpanded ? 'Collapse' : 'Expand'} row`}
-                    size="lg"
```

Comment:
| Before: | After: |
|---------|--------|
| <img width="198" alt="Screenshot 2023-05-03 at 18 43 42" src="https://user-images.githubusercontent.com/48948963/235983756-cae2863c-f06c-4f09-8c95-134b0f56cb58.png">  | <img width="225" alt="Screenshot 2023-05-11 at 10 52 02" src="https://github.com/grafana/grafana/assets/48948963/993de142-7cf2-4ab9-a72c-fb3fd887d783"> |

Changed to `md` since the chevron used above is also size `md`